scrapi/harvesters/osf.py

```diff
@@ -96,9 +95,10 @@ def schema(self):
         }
 
     def harvest(self, start_date=None, end_date=None):
-
-        start_date = start_date or date.today() - timedelta(settings.DAYS_BACK)
```

Comment:
We can if we need to! If start date is passed in that will still be used, this just defaults to 4 days back